### DISCUSSION TITLE
How to list ordinals (numbers) of utxo?

### DISCUSSION THREAD

**Discussion by donnlee** [ordinals/ord#1739] [Q&A]
So a topped-up input has common sats appended to it in the output, iiuc.
How can I list all the ordinals (sat numbers) of a utxo? Is there a ord command?

Ref: issue #1538

**Comment by donnlee** [comment]
Seems it requires ord indexing with --index-sats
How big is the index with --index-sats?  >100GB?
Meanwhile, i am using the search box on ordinals.com

**Reply by hotbroker** [reply]
how,what if this utxo have been transfered many times

**Comment by raphjaph** [comment]
Yes build with `--index-sats` but you will need a lot of RAM and about 100gb of disk space.

**Reply by hotbroker** [reply]
so is there anyway command to do that ,or can i get the sats number from a given utxo?
